### PR TITLE
Rename actuator field to profile (issue #398)

### PR DESCRIPTION
**Pull request by davaya**

### CODE REVIEW

**Comment by dlemire60** [comment]
Addresses issue #398.  I reviewed this on 8/18/2022 and have no comments. Will continue to observe if / as more commits are added.

**Comment by Vasileios-Mavroeidis** [comment]
The changes look correct and accurate based on what we have discussed.

Language recommendation:
Actuator Profile: "An OpenC2 Specification that defines a category...."

**Comment by dlemire60** [comment]
This issue also relates to Architecture Specification [Issue 67](https://github.com/oasis-tcs/openc2-oc2arch/issues/67), and a complementary PR will likely be needed for the Architecture Specification once this change is finalized.

**Comment by dlemire60** [comment]
Presented at the 8/24/2022 working meeting.  Notes:

- @dlemire60 suggested that the **Actuator Profile** definition should emphasize the functional content, not the document, and agreed to propose an updated definition.
- @davaya will add a definition for **Actuator Profile Identifier** to the glossary, since that term is introduced in new text in this PR
- Patrick Maroney asked about the remove of a MUST from (old numbering) lines 869-870; @davaya explained that the MUST was implicit in the way the property tables specify the language and so having it in text seemed superfluous.
- The change from **`actuator`** to **`profile`** in the message (new line 1178) is the heart of the breaking change
- The list of APs in section 3.3.1.3 _is_ the normative list; the text also points to the registry in the Architecture Spec repo but this normative list in the LS aligns with @sparrell's preference to capture this in the LS and publish an updated version when the list changes.

File: oc2ls.md [hunk 1]
```diff
@@ -316,35 +316,37 @@ Intellectual Property Rights section of the TC's web page
...
 
--   **Action**: The task or activity to be performed (e.g., 'deny').
+- **Action**: The task or activity to be performed (e.g., 'deny').
 
--   **Actuator**: The function performed by the Consumer that executes the
-    Command (e.g., 'Stateless Packet Filtering').
+- **Actuator**: The Consumer that executes a Command.
 
--   **Argument**: A property of a Command that provides additional information
+- **Actuator Profile**: The document that defines a category of operations
```
Comment (dlemire60) [inline oc2ls.md:323]: Proposed definition:  
**Actuator Profile**: A defined subset of the OpenC2 language (i.e., actions, targets, command arguments, results) plus any extensions required to specify the use of OpenC2 to command a particular function. Actuator Profiles are documented in published specifications and can be represented as schemas.

File: oc2ls.md [hunk 2]
```diff
@@ -2226,6 +2258,8 @@ requested in the Command.
     "status": 102
 }
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+              
+*Editor's Note: Replace with an example that does not use "properties".*
```
Comment (dlemire60) [inline oc2ls.md:2262]: PR #401 deletes this example entirely. Some other example may be worth adding in its place.

File: oc2ls.md (hunk 1 above)
Comment (Vasileios-Mavroeidis) [reply oc2ls.md:323]: +1 for the definition of ap that @dlemire60 proposed

**Comment by dlemire60** [comment]
Approved for merging at the 7 September 2022 working meeting.

**Pull request merged**